widget/view.controller.js

```diff
@@ -114,14 +117,14 @@
         innerTxt.setAttribute('title', $scope.config.moduleList[i].title)
 
         var count = document.createElement('div');
+        count.setAttribute('id', $scope.config.wid+'layer-'+(i+1)+"-count")
```

Comment:
Please feel free to add code comments to describe the logic, where ever possible